docs/_includes/Expressions.md

```diff
@@ -27,18 +27,27 @@ in order of increasing binding power.
  `in`                     | collection membership
  `!in`                    | collection non-membership
  `!!`                     | disjointness
+--------------------------|------------------------------------
+ `<<`                     | left-shift
+ `>>`                     | right-shift
 --------------------------|------------------------------------
  `+`                      | addition (plus)
  `-`                      | subtraction (minus)
 --------------------------|------------------------------------
  `*`                      | multiplication (times)
  `/`                      | division (divided by)
  `%`                      | modulus (mod)
+--------------------------|------------------------------------
+ `|`                      | bit-wise or
+ `&`                      | bit-wise and
+ `^`                      | bit-wise exclusive-or (not equal)
 --------------------------|------------------------------------
  `-`                      | arithmetic negation (unary minus)
- `!`, &not;               | logical negation
+ `!`                      | logical negation, bit-wise complement
  Primary Expressions      |
 
+TODO: What about unicode symbols for ! & | <==> ==> <==
```

Comment:
Actually, I'd rather stay away from the Unicode symbols for all operators. They're a pain to find on a keyboard, and IDEs are better at converting ASCII characters into pretty symbols than humans are at finding the right Unicode symbol to enter. So, I'd even be happy if we removed the Unicode variants of the existing operators from the `.atg` file.

Reply:
OK